### PR TITLE
First Changes

### PR DESCRIPTION
**Pull request by jonathanmunamire**
This is my first pull request on this project

You Can Check the Recorded Presentation Video of the Project on This [Loom link](https://www.loom.com/share/a416abdd6e8547a389c19b11e6727fd4)
You can Take a Look at This Project on This [Link](https://jonathanmunamire.github.io/Polyglot/)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    <title>Polyglot</title>
+    <link rel="icon" href="IMG/logo.jpg" type="image/x-icon">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
```
Comment (OLIPLICHE) [inline about.html:17]: # Please **Mobile size** has a white background and same color. 
- See your design: pay attention to the color
- 
Capture
- See original design:
- 
Capture

File: index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    <title>Polyglot</title>
+    <link rel="icon" href="IMG/logo.jpg" type="image/x-icon">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
```
Comment (OLIPLICHE) [inline index.html:17]: # Desktop Size
- **Home Page**: It has a white background and same color.
- See your design
- 
Capture
- See the original design
- 
Capture
- **About page** is a dark background. 
- See original design:
- 
Capture

File: about.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                <h2 class="footer-heading">Partners</h2>
+                <hr class="line3">
+                <div class="partners">
+                    <p>KAHOOT</p>
+                    <p>Udemy</p>
+                    <p>COURSERA</p>
+                    <p>Stride</p>
+                    <p>Unacademy</p>
+                    <div>
+            </section>
```
Comment (OLIPLICHE) [inline about.html:104]: - Please **About page** does not contain the partners' section, Kindly remove it. This applies to About page Mobile size as well
- See the original design
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @jonathanmunamire, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Deploy the web pages in GitHub👏 👏
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
- Please provide a better online link of the uploaded video. The link which you shared is not working.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Please I provided the Link to the 👉 [Original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                <a class="menu-item" href="about.html">About</a>
+            </div> 
+        </nav>
+    </header>
+
+    <main>
+        <section class="first-section" id="home">
+            <div class="greet">
+                <h2 class="greet1">Interested in Languages?</h2>
+                <h2 class="greet2">POLYGLOT LANGUAGES SCHOOL <br>We're here for you! <br>2022</h2>
```
Comment (leonardodiasb) [inline index.html:46]: - [OPTIONAL] There should be at least 1, but no more than 1, `h1` tag on each page for the main title and this looks like the main title of this page. Would be nice if you change that tag for semantic purposes and better SEO.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                <a class="menu-item" href="about.html">About</a>
+                <a class="menu-item" href="#logo">Logo</a>
+                <a class="menu-item" href="#vision">Vision</a>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="first-section" id="home">
+            <div class="greet">
+                <h2 class="greet2">POLYGLOT LANGUAGES SCHOOL <br>KIGALI (RWANDA)</h2>
```
Comment (leonardodiasb) [inline about.html:44]: - [OPTIONAL] There should be at least 1, but no more than 1, `h1` tag on each page for the main title and this looks like the main title of this page. Would be nice if you change that tag for semantic purposes and better SEO.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    <header>
+        <nav class="header1">
+            <a href="#">facebook</a>
+            <a href="#">twitter</a>
+            <a href="#">english</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="header2">
+            <a class="logo" href="index.html"><img src="IMG/logo.jpg" height="50vh"></a>
```
Comment (leonardodiasb) [inline about.html:27]: - [OPTIONAL] Please make sure the page is accessible, images should contain `alt` attributes for accessibility purposes.

File: about.html [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+            <p class="address">@243 East Section, Hotel2000, Kigali (Rwanda)</p>
+        </section>
+        <section class="second-section" id="logo">
+            <h2>Polyglot Languages School Logo</h2>
+            <hr class="line2">
+            <p>The Logo of Polyglot Languages School was decided through the logo competition from 8.June to 7.July.
+                After online public voting and Selection Committee's judjing process, the logo from a 17years old
+                Rwandan girl won the prize. After consultation with Mr. Jonathan Amani Munamire we optimized the
+                prize-winning-logo for Polyglot Language School as following:</p>
+            <img src="IMG/logo.jpg">
```
Comment (leonardodiasb) [inline about.html:68]: - [OPTIONAL] Please make sure the page is accessible, images should contain `alt` attributes for accessibility purposes.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+    <header>
+        <nav class="header1">
+            <a href="#">facebook</a>
+            <a href="#">twitter</a>
+            <a href="#">english</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="header2">
+            <a class="logo" href="index.html"><img src="IMG/logo.jpg" height="50vh"></a>
```
Comment (leonardodiasb) [inline index.html:27]: - [OPTIONAL] Please make sure the page is accessible, images should contain `alt` attributes for accessibility purposes.

File: script.js [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+},
+];
+
+const card = document.querySelector('.teacher-cards');
+
+Teachers.forEach((cards) => {
+    const div = document.createElement('div');
+    div.innerHTML += `
+  <div class="card2">
+  <img src="${cards.teacherImage}">
```
Comment (leonardodiasb) [inline script.js:57]: - [OPTIONAL] Please make sure the page is accessible, images should contain `alt` attributes for accessibility purposes.

File: script.js [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+  <div>
+      <h3>${cards.teacherName}</h3>
+      <h5>${cards.teacherDomain}</h5>
+      <p>${cards.teacherAbout}</p>
+  </div>
+  </div>
```
Comment (leonardodiasb) [inline script.js:63]: - [OPTIONAL] The 'Featured Speakers' section is a list of speakers, so probably wrapping the list with a `ul` tag and each speaker with the `li` tag would be better for semantics.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @jonathanmunamire 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - README file is descriptive 👏
 - Commit messages are meaningful 👏 👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 5 above)
Comment (jonathanmunamire) [reply about.html:44]: Great, I will implement that

File: about.html (hunk 6 above)
Comment (jonathanmunamire) [reply about.html:27]: Great, I will implement that

File: about.html (hunk 7 above)
Comment (jonathanmunamire) [reply about.html:68]: Great, I will implement that

File: index.html (hunk 8 above)
Comment (jonathanmunamire) [reply index.html:27]: Great, I will implement that

File: script.js (hunk 9 above)
Comment (jonathanmunamire) [reply script.js:57]: Great, I will implement that

File: script.js (hunk 10 above)
Comment (jonathanmunamire) [reply script.js:63]: Great, I will implement that

**Comment by jonathanmunamire** [comment]
Thank you so much @leonardodiasb for your review, I appreciate it. I will certainly take your feedback into consideration and make the necessary changes to improve my code.

**Pull request merged**